cla_backend/apps/cla_eventlog/management/commands/find_and_delete_old_cases.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand
+from dateutil.relativedelta import relativedelta
+
+from legalaid.models import Case
+from cla_butler.tasks import DeleteOldData
+
+
+class FindAndDeleteCasesUsingCreationTime(DeleteOldData):
+    def get_eligible_cases(self):
+        super(FindAndDeleteCasesUsingCreationTime, self)._setup()
```

Comment:
I don't think you want to be calling `_setup()` via `super` here.
`super` is generally used for calling the version of the method you're in, from a parent.
eg if we wanted to run the parent's version of `get_eligible_cases()` but then filter the result, we would use `super()` to let us call that parent version of `get_eligible_cases()` from inside `get_eligible_cases()`.
If we want to just call another method on the class, _even if it was defined by a parent class_, we can just call it. `FindAndDeleteCasesUsingCreationTime` will have all the same methods on it as `DeleteOldData`, we don't generally have to worry (or even know!) which were defined on a parent.

There _are_ reasons to call a different method with `super`, but they're not usual.

Reply:
Thanks again I forgot I could use `self` lol